### ISSUE TITLE
On a new borrow getting 406 error with default odm file download

### ISSUE THREAD

**Issue by jvandervort** [ping/odmpy#37]
On a new borrow getting 406 error with default odm file download.
Not sure if this is my library no longer allowing an odm file access or something else, but I thought I'd report it.
I had an old borrow in my loans, it worked, but both new borrows had this error.
NOTE:  --direct worked fine

commit 5af3caceb6c4a2e65be4e7dc703674bd89b3be42

```
An unexpected error has occured
Traceback (most recent call last):
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\libby.py", line 349, in make_request
    res.raise_for_status()
  File "C:\Users\nnn\AppData\Roaming\Python\Python311\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 406 Client Error: Not Acceptable for url: https://sentry-read.svc.overdrive.com/card/35351901/loan/74599/fulfill/LibbyFormats.AudioBookMP3

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\odm.py", line 1035, in run
    extract_loan_file(libby_client, selected_loan, args),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\odm.py", line 358, in extract_loan_file
    odm_res_content = libby_client.fulfill_loan_file(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\libby.py", line 683, in fulfill_loan_file
    res: requests.Response = self.make_request(
                             ^^^^^^^^^^^^^^^^^^
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\libby.py", line 358, in make_request
    ErrorHandler.process(http_err)
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\libby_errors.py", line 103, in process
    raise ClientError(
odmpy.libby_errors.ClientError: <odmpy.libby_errors.ClientError; http_status=406, msg='406 Client Error: Not Acceptable for url: https://sentry-read.svc.overdrive.com/card/35351901/loan/74599/fulfill/LibbyFormats.AudioBookMP3', error_response=' ''>
Traceback (most recent call last):
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\libby.py", line 349, in make_request
    res.raise_for_status()
  File "C:\Users\nnn\AppData\Roaming\Python\Python311\site-packages\requests\models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 406 Client Error: Not Acceptable for url: https://sentry-read.svc.overdrive.com/card/35351901/loan/74599/fulfill/LibbyFormats.AudioBookMP3

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\__main__.py", line 34, in <module>
    main()
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\__main__.py", line 26, in main
    run()
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\odm.py", line 1035, in run
    extract_loan_file(libby_client, selected_loan, args),
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\odm.py", line 358, in extract_loan_file
    odm_res_content = libby_client.fulfill_loan_file(
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\libby.py", line 683, in fulfill_loan_file
    res: requests.Response = self.make_request(
                             ^^^^^^^^^^^^^^^^^^
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\libby.py", line 358, in make_request
    ErrorHandler.process(http_err)
  File "C:\Users\nnn\Documents\dev\odmpy\odmpy\libby_errors.py", line 103, in process
    raise ClientError(
odmpy.libby_errors.ClientError: <odmpy.libby_errors.ClientError; http_status=406, msg='406 Client Error: Not Acceptable for url: https://sentry-read.svc.overdrive.com/card/35351901/loan/74599/fulfill/LibbyFormats.AudioBookMP3', error_response=' ''>

```

**Comment by ping** [comment]
I *think* this should be fixed in master, can you refresh your copy and try it again?

**Comment by jvandervort** [comment]
Latest head now works to download, but also does not include track numbers.

**Comment by ping** [comment]
The track number issue is addressed in #36, so I'll close this one off as fixed.
